core/procedure.h

```diff
@@ -33,6 +33,10 @@ struct Procedure;
 
 typedef EntityType<Procedure> ProcedureType;
 
+static std::string GENERIC_OUTPUT_DS_DESC =
```

Comment:
constexpr const char * would be a better type I think because the string should be constant and shared.